Lesson 06 integrate components

Deman4ig opened the pull request.

Deman4ig committed:
Add solution for lesson 06: add component previews

Deman4ig committed:
Fix background color for header and footer

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

Deman4ig committed:
Add preview modificators to stretch components

jsru commented:
Решение было обновлено, посмотрим что скажет @GSemikozov

GSemikozov commented on components/card/card.css:
```diff
@@ -1,14 +1,24 @@
-    justify-content: space-between;
+    padding: 32px 25px;
+    box-sizing: border-box;
+    height: 100%;
```
точно ли нужно всегда по умолчанию иметь высоту в 100%?

Deman4ig replied:
эту высоту добавил чтоб делать карточки в ряду одинаковой высоты, т.к. иначе не получалось сделать красоту.

но сейчас вроде попробовал по другому и убрал один див лишний из всех карточек

GSemikozov replied:
да, как раз к этому и был вопрос)

GSemikozov commented on components/card/card.css:
```diff
@@ -1,14 +1,24 @@
+    flex-grow: 1;
+}
+
+.card__body-gap {
```
тут это скорее модификатор - .card__body_gap

Deman4ig replied:
Поправил

Deman4ig committed:
Refactor card styles

1. Rename card__body_gap modificator
2. Remove fixed height from card component
